src/controllers/keyboard/keyboardeventfilter.h

```diff
@@ -6,24 +6,27 @@
 #include <QKeyEvent>
 #include <QMultiHash>
 
-#include "preferences/configobject.h"
+#include "controllers/keyboard/keyboardcontrollerpreset.h"
 
 class ControlObject;
 
 // This class provides handling of keyboard events.
 class KeyboardEventFilter : public QObject {
     Q_OBJECT
   public:
-    KeyboardEventFilter(ConfigObject<ConfigValueKbd> *pKbdConfigObject,
-                        QObject *parent = nullptr, const char* name = nullptr);
+    KeyboardEventFilter(QObject* parent = nullptr, const char* name = nullptr);
```

Comment:
This also counts as a single-parameter constructor and is used for implicit type conversion from QObject* to KeyboardEventFilter if not declared 'explicit'!